hummingbot/strategy/cross_exchange_market_making/cross_exchange_market_making.pyx

```diff
@@ -1163,7 +1166,9 @@ cdef class CrossExchangeMarketMakingStrategy(StrategyBase):
                 taker_trading_pair, True, quote_asset_amount
```

Comment:
Since we actually already have the active_order price, I think we can use that instead of recalculating the price instead 
`taker_price = order_price`

Reply:
sorry, my bad, the check_and_hedge function uses the following method when calculating the price to hedge so we should use the same in order not to result in insufficient balance.
```
            order_price = taker_market.get_price_for_volume(taker_trading_pair, True,
                                                            quantized_hedge_amount).result_price
```
so we should probably change this to calculate based on the actual order quantity instead of the quote asset amount which is a much larger sum.

`taker_price = taker_market.get_price_for_volume(taker_trading_pair, True, active_order.quantity).result_price`